bigquery/google/cloud/bigquery/job.py

```diff
@@ -1384,20 +1368,39 @@ def from_api_repr(cls, resource):
     https://g.co/cloud/bigquery/docs/reference/rest/v2/jobs#configuration.query.createDisposition
     """
 
-    default_dataset = _TypedApiResourceProperty(
-        'default_dataset', 'defaultDataset', DatasetReference)
-    """See
-    https://g.co/cloud/bigquery/docs/reference/v2/jobs#configuration.query.defaultDataset
-    """
+    @property
+    def default_dataset(self):
+        """google.cloud.bigquery.dataset.DatasetReference: the default dataset
```

Comment:
This renders as a link to the property's class?

Reply:
These don't render as links to the class. That line ends up getting formatted like this:

_google.cloud.bigquery.dataset.DatasetReference_ – the default dataset to use for unqualified table names in the query.

I can make it into a link by adding `:class:` before it, but that would depart from the Sphinx Google style [example](http://www.sphinx-doc.org/en/stable/ext/example_google.html).